src/main/scala/dpla/ingestion3/model/DplaMapData.scala

```diff
@@ -139,8 +138,7 @@ case class DplaPlace(
                      county: ZeroToOne[String] = None,
                      state: ZeroToOne[String] = None,
                      country: ZeroToOne[String] = None,
-                     // Region or equivalent is not returned by Twofishes
-                     // region: ZeroToOne[String] = None,
+                     region: ZeroToOne[String] = None,
```

Comment:
I'm laying this out in a order sensitive `Row` as you mention below, so I had to basically bake in a somewhat final representation of everything, and needed to make some choices.

I don't think there was a good reason to remove this field previously... it's optional, it's in the MAP spec, and could easily be satisfied with a lookup dictionary of country -> region (i.e., we don't need Twofishes to support it for it to have a reason to exist and a way for it to be populated). So I put it back in so it could sit in the natural order of small to big fields and we wouldn't have to do any complicated field shuffling to get it back.